src/worker/utils/chem.ts

```diff
@@ -136,6 +136,101 @@ export const getMatchingSubstructure = ({ structure, substructure }: { structure
   return { matchingAtoms: atoms, matchingBonds: bonds };
 };
 
+export const isValidMolBlock = (mdl: string) => {
+  if (!mdl.includes('M  END')) return false;
+  const mol = get_molecule(mdl, globalThis.workerRDKit);
+  if (!mol) return false;
+  try {
+    return mol.is_valid();
+  } finally {
+    release_molecule(mol);
+  }
+};
+
+/**
+ * Convert molecule structure to given notation (smiles, smarts, molblock, ...)
+ * TODO some notations like inchi don't work with qmol
```

Comment:
is this still TODO or is this comment stale?

Reply:
Changed to warning. We just need to avoid using qmol to convert to aromatic_form (throw error) and inchi (return empty string). I will try to solve this using TS in the future.